DuckDuckGoTests/AppUserDefaultsTests.swift

```diff
@@ -166,15 +166,9 @@ class AppUserDefaultsTests: XCTestCase {
         XCTAssertEqual(appUserDefaults.autofillCredentialsEnabled, false)
     }
 
-    func testDefaultAutoconsentStateIsFalse_WhenNotInRollout() {
+    func testDefaultAutoconsentStateIsTrue() {
         let appUserDefaults = AppUserDefaults(groupName: testGroupName)
-        appUserDefaults.featureFlagger = createFeatureFlagger(withSubfeatureEnabled: false)
-        XCTAssertFalse(appUserDefaults.autoconsentEnabled)
-    }
 
-    func testDefaultAutoconsentStateIsTrue_WhenInRollout() {
-        let appUserDefaults = AppUserDefaults(groupName: testGroupName)
-        appUserDefaults.featureFlagger = createFeatureFlagger(withSubfeatureEnabled: true)
         XCTAssertTrue(appUserDefaults.autoconsentEnabled)
     }
 
```

Comment:
Not sure if it makes sense to keep this test `testAutoconsentReadsUserStoredValue_RegardlessOfRolloutState`?

Reply:
True! Removed in https://github.com/duckduckgo/iOS/pull/2537/commits/959f8f734035df45d462c9c299a795e102faa1cb.